requirements/web-api

```diff
@@ -0,0 +1,12 @@
+# Please make sure to pin all dependency versions, in order to avoid unwanted
+# functional and integration breaks caused by external code updates.
+#
+# These are web-api dependencies, so pin them.
```

Comment:
> This comment does not seem to provide a lot of additional clarity, at least to me 😅

Reply:
Agreed, I guess pinning only makes sense for dev deps.

Reply:
I was really focusing on the comment itself more than on arguing the validity of pinning those dependencies :)

Reply:
Added a better comment, let me know if it's ok.